packages/itwinui-react/src/core/Tree/Tree.tsx

```diff
@@ -159,6 +163,7 @@ export const Tree = <T,>(props: TreeProps<T>) => {
     className,
     nodeRenderer,
     getNode,
+    size,
```

Comment:
Should `size="default"` be set as the default? It works without it too, but would this be better for consistency?

Reply:
sure

```suggestion
    size = 'default',
```